Automattic/crowdsignal-ui
# Include Navigation Bar and settings on the Editor

## Summary

This PR adds a few settings which allow showing a navigation bar in the Projects.
Those changes affect only the Editor for now.

Depends on: D84663-code

## Testing Instructions
* Apply backend changes
* Open or create a new project
* Open the Project settings on the right sidebar
* You should see a `Navigation` section
* Changing the toggles should allow you to customize the way you want to see the Navigation Bar
